File: js/OOP/methods.js
// create blueprint

class Person {
    constructor(nameOfPerson, ageOfPerson) {
        this.name = nameOfPerson;
        this.age = ageOfPerson;
    }

    greet() {
        console.log(`Hello ${this.name}`)
    }

    incrementAge() {
        this.age = this.age + 1;
    }
}
// create the object from the blueprint
const p1 = new Person('shaked1', 23)
p1.greet()
console.log(p1.age)
p1.incrementAge()
console.log(p1.age)

File: python/OOP/methods.py
# create blueprint
class Person :
    def __init__(self, nameOfPerson, ageOfPerson):
        self.name = nameOfPerson
        self.age = ageOfPerson

    def greet(self):
        print(f'Hello {self.name}')

    def incrementAge(self):
        self.age = self.age + 1

# create the object from the blueprint
p1 = Person('y', 24)
print(p1.age)
p1.incrementAge()
print(p1.age)
p1.greet()